src/components/Navbar.jsx

```diff
@@ -4,7 +4,7 @@ import { Link } from 'react-router-dom';
 const Navbar = () => (
   <nav className="navBar">
     <div className="navItems">
-      <h2>Supreme Bookstore</h2>
+      <h2 className='navTitle'>Supreme Bookstore</h2>
```

Comment:
- [ ] Your navbar looks really great. Just to be 100% aligned with the original design, Please use something similar for the user icon including the color, and apply a light border around your user icon.
- [ ] Also, Notice the active page is a bit bolder than the inactive one, Please address this too. 

### Your design 
![image](https://github.com/Cisco-the-wayword/Bookstore/assets/69077061/0a501c15-c289-4a87-ba5a-fe868ca70167)
### Original design 
![image](https://github.com/Cisco-the-wayword/Bookstore/assets/69077061/7aa1981c-e41f-44df-9fbb-69e120ca798e)